src/constants/StyleConstants.ts

```diff
@@ -1,10 +1,12 @@
 const basicGrey = "rgb(236,236,236)";
 const trueWhite = "white";
+const niceBlue = "rgba(45, 65, 123, 1)";
```

Comment:
Can we use one of these? https://github.com/CouncilDataProject/cdp-design/blob/main/generate-cdp-colors.js#L3

I would say dark blue or neon-green or something?